ballerina/tests/client_endpoint_negative_test.bal

```diff
@@ -189,3 +189,28 @@ public function testRemoveDirectoryWithWrongUrl() {
         test:assertFail(msg = "Found a non-error response while accessing a non-existing directory path.");
     }
 }
+
```

Comment:
```suggestion
Disabling due to https://github.com/ballerina-platform/ballerina-standard-library/issues/2703
```

Reply:
Updated in https://github.com/ballerina-platform/module-ballerina-ftp/pull/549/commits/6d021f9a4d6893d782f721a9a92825e2a19b0c0e